core/src/main/scala/cromwell/core/path/DefaultPathBuilderFactory.scala

```diff
@@ -3,6 +3,8 @@ package cromwell.core.path
 import akka.actor.ActorSystem
 import cromwell.core.WorkflowOptions
 
+import scala.concurrent.{ExecutionContext, Future}
+
 case object DefaultPathBuilderFactory extends PathBuilderFactory {
-  override def withOptions(options: WorkflowOptions)(implicit actorSystem: ActorSystem) = DefaultPathBuilder
+  override def withOptions(options: WorkflowOptions)(implicit as: ActorSystem, ec: ExecutionContext) = Future.successful(DefaultPathBuilder)
```

Comment:
Oh I meant is there a reason the variable name was changed from `actorSystem` to `as`?